web/src/components/netflow-traffic.tsx

```diff
@@ -245,7 +245,7 @@ export const NetflowTraffic: React.FC<NetflowTrafficProps> = ({ forcedFilters, i
   const [lastTop, setLastTop] = useLocalStorage<number>(LOCAL_STORAGE_LAST_TOP_KEY, TOP_VALUES[0]);
   const [range, setRange] = React.useState<number | TimeRange>(getRangeFromURL());
   const [histogramRange, setHistogramRange] = React.useState<TimeRange>();
-  const [metricScope, setMetricScope] = useLocalStorage<FlowScope>(LOCAL_STORAGE_METRIC_SCOPE_KEY, 'namespace');
+  const [metricScope, _setMetricScope] = useLocalStorage<FlowScope>(LOCAL_STORAGE_METRIC_SCOPE_KEY, 'namespace');
```

Comment:
We have existing cases like this:
- updateTopologyMetricType
- updateTableFilters

I think we should decide which approach to use and I can adopt it in [NETOBSERV-268](https://issues.redhat.com/browse/NETOBSERV-268)

Reply:
Do you mean only for the naming, or something else? I can rename `setMetricScope` / `updateMetricScope`